tsl/src/continuous_aggs/common.c

```diff
@@ -432,13 +387,11 @@ cagg_agg_validate(Node *node, void *context)
 static bool
 cagg_query_supported(const Query *query, StringInfo hint, StringInfo detail, const bool finalized)
 {
-/*
- * For now deprecate partial aggregates on release builds only.
- * Once migration tests are made compatible with PG15 enable deprecation
- * on debug builds as well.
- */
-#ifndef DEBUG
-#if PG15_GE
+	/*
+	 * For now deprecate partial aggregates on release builds only.
```

Comment:
I guess this comment can be removed now.

Reply:
Removed!